packages/app/package.json

```
@@ -130,7 +130,7 @@
     "@babel/preset-env": "^7.4.2",
     "@cerebral/http": "^4.0.0",
     "@cerebral/mobx-state-tree": "^3.0.0",
-    "@codesandbox/template-icons": "^0.3.0",
+    "@codesandbox/template-icons": "^0.4.0",
```

Latest version has the Unibit icon

```suggestion
    "@codesandbox/template-icons": "^0.5.0",
```